assets/src/apps/v2/elevator.tsx

```diff
@@ -25,8 +25,8 @@ import NoData from "Components/v2/elevator/no_data";
 const TYPE_TO_COMPONENT = {
   normal: NormalScreen,
   takeover: TakeoverScreen,
-  elevator_closures_list: ElevatorClosuresList,
-  current_elevator_closed: CurrentElevatorClosed,
+  elevator_closures: ElevatorClosures,
+  elevator_closed: ElevatorClosed,
```

Comment:
An alternative could be something like `UnavailableElevators` for the list of all other elevators that are closed? That makes the `ElevatorClosed` component seem clearer and more distinct to me. 

Also like the word outages, could be something like `AllElevatorOutages`, or even just `AllElevatorClosures`. I think that naming could then be confusing in the future then if we do narrow down which closures are shown at each elevator

Reply:
To follow up, I do really like the spirit of this PR, but agree that `ElevatorClosures` and `ElevatorClosed` are too similar. 24 hours to let it marinate though, and my new idea is incorporating that when the current elevator is closed, we include the alternate route map. Could be `ElevatorAlternateRoute` and `ElevatorClosures`

Reply:
I went with `ElevatorAlternatePath` to avoid semantic overlap with `Route` as in a GTFS route, but otherwise took that suggestion. What do you think?

Reply:
looking through the code with these changes, I really like it! Will be much clearer in the future.